Makefile

```diff
@@ -18,6 +18,9 @@ SOURCE := $(wildcard src/*.c)
 
 EXECUTABLE := uvb-server
 
+DB_PREFIX := /var/db
```

Comment:
For now, lets just keep the prefix set to the current directory. I'm planning on making the db location changeable from the cli some time in the near future, and then we can better discuss default locations.